### PR TITLE
Raise an exception if an annotated Algorithm has no @Execute annotation

### LINKED ISSUE #164: Validate AbstractAnnotatedAlgorithm
Currently, if no method is tagged `@Execute` in a class extending `AbstractAnnotatedAlgorithm`, the program will fail with a `NullPointerException` at execution time (see the [line of code](https://github.com/52North/WPS/blob/dev/52n-wps-algorithm/src/main/java/org/n52/wps/server/AbstractAnnotatedAlgorithm.java#L54)). It would be nice for this to get validated when algorithm is loaded instead.

Cheers

### PR DESCRIPTION
**Pull request by htulipe**
This PR fixes #164

### CODE REVIEW

File: 52n-wps-algorithm/src/main/java/org/n52/wps/algorithm/annotation/AnnotatedAlgorithmIntrospector.java [hunk 1]
```diff
@@ -180,7 +180,11 @@ private void parseClass() {
                 }
             }
         }
-        
+
+        if(this.executeMethodBinding == null) {
+            throw new RuntimeException("No execute method binding for class " + getClass().getCanonicalName());
```
Comment (tkunicki) [inline 52n-wps-algorithm/src/main/java/org/n52/wps/algorithm/annotation/AnnotatedAlgorithmIntrospector.java]: I think this reports the name of the interpreter and not the subject class (`algorithmClass`)

Comment (htulipe) [inline 52n-wps-algorithm/src/main/java/org/n52/wps/algorithm/annotation/AnnotatedAlgorithmIntrospector.java]: You might be right, I blindly copy/paste this from a couple lines above. I'll change that.

**Comment by htulipe** [comment]
About the licence agreement, I don't have a printer at home and if I could avoid sending document abroad I would appreciate. Is there a way to send a signed PDF via email?

**Comment by htulipe** [comment]
Okay, I updated the code.

**Comment by nuest** [comment]
Regarding the singed PDF: afaik there is now way around a signature on paper, I'm afraid. @ahitch an we help out @htulipe ? If you send your address details to cla@52north.org we could send you a paper copy for you to sign and send back...

**Comment by htulipe** [comment]
Thanks for the offer @nuest but if no solution is possible via email, I will find a printer don't worry.

**Comment by nuest** [comment]
I am really sorry for the burocracy and paperwork - I am also checking with our legal experts if small submissions might go in without CLA. If you eventually do sign a CLA maybe that's a good motivation to submit more of these very useful code improvements!

On that note @bpross-52n : Code looks fine to me, let's wait for @ahitch @matthesrieke @wytzisk to get back to us on the legal stuff.

**Comment by nuest** [comment]
The verdict is in: no way around the signed paper - you'll have to print, sign, and send us two copies of the CLA. Thanks for the understanding, and I hope the hassle pays off in many upcoming helpful bugfixes like this one!

Let me know if you have any further questions or get in touch with cla@52north.org - Thanks!

**Comment by htulipe** [comment]
Okay I will send two copies of the CLA via standard mail.

**Pull request merged**